src/main/webapp/package-lock.json

```diff
@@ -0,0 +1,13022 @@
+{
```

Comment:
Can this file be removed now since we are back to using yarn?

Reply:
Yup, thanks

Reply:
Removed in 3c87534fb1ab199418e56816bb7eef2a5f1bf12a